rs/recovery/src/file_sync_helper.rs

```diff
@@ -85,12 +86,25 @@ pub fn rsync_with_retries(
             require_confirmation,
             key_file,
         ) {
-            Err(e) => warn!(logger, "Rsync failed: {:?}, retrying...", e),
+            Err(e) => {
+                warn!(logger, "Rsync failed: {:?}", e);
+                if auto_retry {
+                    // In non-interactive cases, we wait a short while
+                    // before re-trying rsync.
+                    info!(logger, "Retrying in 10 seconds...");
+                    std::thread::sleep(std::time::Duration::from_secs(10));
```

Comment:
nit: we already import `std::time::Duration`

```suggestion
                    std::thread::sleep(Duration::from_secs(10));
```

Reply:
done